### PR TITLE
Refactor/clients folder

### PR DESCRIPTION
**Pull request by Cerchie**
## Summary of Changes

Moving AdminClients, KafkaProducerClients, and SchemaRegistry clients to their own `clients` folder and updating some imports. Had to change some methods in Clients to protected to make tests pass.

## Any additional details or context that should be provided?

Refactoring in preparation for addressing #153 

## Pull request checklist

Please check if your PR fulfills the following (if applicable):

- [x] Have you validated this change locally against a running instance of the Quarkus dev server?
    ```shell
    make quarkus-dev
    ```
- [x] Have you validated this change against a locally running native executable?
    ```shell
    make mvn-package-native && ./target/ide-sidecar-*-runner
    ```

### CODE REVIEW

File: src/main/java/io/confluent/idesidecar/restapi/messageviewer/MessageViewerContext.java [hunk 1]
```diff
@@ -10,7 +10,6 @@
 import io.vertx.core.http.HttpMethod;
 import java.util.Map;
 import java.util.Optional;
-
```
Comment (rhauch) [inline src/main/java/io/confluent/idesidecar/restapi/messageviewer/MessageViewerContext.java:13]: We should maintain one blank line between imports and the class definition.

File: src/main/java/io/confluent/idesidecar/restapi/messageviewer/MessageViewerClusterInfoProcessor.java [hunk 2]
```diff
@@ -74,7 +74,6 @@ public Future<MessageViewerContext> process(MessageViewerContext context) {
       );
     }
   }
-
```
Comment (rhauch) [inline src/main/java/io/confluent/idesidecar/restapi/messageviewer/MessageViewerClusterInfoProcessor.java:77]: We should maintain one blank line between methods.

**Review by rhauch — COMMENTED** [review]
Quick work, @Cerchie. Two minor nits, but should be good once those are fixed.

**Comment by rhauch** [comment]
Looks like there are some build errors related to the `ClientConfigurator` that was recently added. It should probably also be moved to the `clients` package.

**Comment by Cerchie** [comment]
> Looks like there are some build errors related to the `ClientConfigurator` that was recently added. It should probably also be moved to the `clients` package.

moved! heads up, looks like my defaults added some style changes in terms of spacing etc that I didn't catch in the diff. Happy to revert if needed.

**Comment by Cerchie** [comment]
closing to open a fresh one without auto-formats and conflicts

**Pull request closed without merging**